packages/manager/modules/pci/src/projects/project/training/jobs/info/logs/logs.html

```diff
@@ -8,17 +8,7 @@
     data-translate="pci_projects_project_training_job_logs_text_3"
 ></p>
 
-<div data-ng-if="$ctrl.currentJob.isTerminal()">
-    <p data-translate="pci_projects_project_training_job_logs_text_5"></p>
```

Comment:
**nitpick(blocking):** Following translation can be removed then from `packages/manager/modules/pci/src/projects/project/training/jobs/info/translations/Messages_fr_FR.json`.

Reply:
Mark as unresolved. Translations are still there. Could you please remove them?